.gitignore

```diff
@@ -1,13 +1,20 @@
-# Directories
+# Binaries
 .gradle/
```

Comment:
All these changes seem not to be correct.

A lot of necessary things are no longer being ignored, for instance the `dist/` directory and `testLauncher/` directory.

Ideally the launcher is run from the `testLauncher` directory within the IDE, the docs mention the `test` directory which is out of date.

I'd say revert these changes for now and I'll take a look at a solution to the root directory issue in development

Reply:
Reverted ignoring of project-specific folders. Retained ignoring of Gradle folders instead of Maven ones (`build` vs. `target`).